docs/reference/commandline/images.md

```diff
@@ -52,6 +52,35 @@ uses up the `VIRTUAL SIZE` listed only once.
     postgres                  9.3.5               746b819f315e        4 days ago          213.4 MB
     postgres                  latest              746b819f315e        4 days ago          213.4 MB
 
+### Listing images by name and tag
+
+The ``docker images`` command takes an optional ``[REPOSITORY[:TAG]]``
```

Comment:
@thaJeztah I have no idea :sweat_smile:. Long time ago, I probably used a markdown lib. where using simple back-ticks didn't work and I got used to it.. I didn't know it was working with simple back-tick `test` :sweat:… (the lazy box inside of me is pretty happy, one keystroke less to type :smile_cat:)